lib.sh

```diff
@@ -177,12 +177,13 @@ function request_2fa_token() {
   echo "🔄 Logging into AWS..." >&2
 
   # Check if Yubikey or 1password are available
-  if ykman=$( which ykman ); then
-    if [ -z "$( $ykman list )" ]; then
-      echo "🔑 Please insert your yubikey and hit <ENTER>..." >&2
-      read
+  if ykman=$( which ykman ) && if [[ "${SETUP_AWS_HELPER_YKMAN:-0}" -eq 1 ]]; then
```

Comment:
Is it the correct syntax? I'm always confused by bash conditions…

Maybe like this
```suggestion
  if ykman=$( which ykman ) && [[ "${SETUP_AWS_HELPER_YKMAN:-0}" -eq 1 ]]; then
```

Reply:
Derp! Copy/paste error, thanks!